sql/updates/world/2016_02_10_01_world.sql

```diff
@@ -0,0 +1,6 @@
+--
+DELETE FROM `spell_proc_event` WHERE `entry`= 56841;
+INSERT INTO `spell_proc_event` VALUES ('56841', '0', '9', '2048', '2048', '2048', '256', '0', '0', '0', '0');
```

Comment:
Looks OK, but a few notes on file format:
- remove apostrophes ( ' ' ) from numbers, only used for names (text)
- add a LineFeed / NewLine at the end of the file (currently missing)